sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/UpdateCenterDeserializer.java

```diff
@@ -408,6 +409,21 @@ private static void resolveRangeOfRequiredSQVersion(Sonar sonar, List<Version> r
     return splitted;
   }
 
+  private static Version resolveLowVersion(String versionStr, String range) {
+    if (LATEST_KEYWORD.equals(versionStr)) {
+      throw new SonarVersionRangeException(String.format(
+        "Cannot use LATEST keyword at the start of a range in '%s'. Use 'sqVersions=LATEST' instead.",
+        range
+      ));
+    } else if (versionStr.endsWith("*")) {
```

Comment:
no need for `else` when `if` block only throws an exception. 
This remove one level of enclosing and makes code more readable

Reply:
How would we set the exception message? As a var, used in the `throw` statement?

Reply:
I don't understand your question.
I assume your question would be about an impact of removing the else bloc, however I see the exception message passed directly to the constructor, so I'm puzzled

Reply:
Ah, sorry. I now understand I misread your comment. My bad 👍 .